### DISCUSSION TITLE
校外時數怎麼賺

### DISCUSSION THREAD

**Discussion by Mitochondrionjessica** [1101-GS4719-C/Python-Community-GS4719-C#1989] [Q&A]
校外時數大概是服務時數最難取得的項目，首先是校外服務時數的活動本來就不多，再加上申請手續繁雜，根本雪上加霜呀~
之前有參加過淨灘，跟同學一起去海邊撿垃圾挺好玩的，除了回味童年記憶又有戶外教學的感覺，還能讓海灘更乾淨，實在是一舉數得，唯一美中不足的就是交通不便QQ
想請問各位都如何賺取校外時數呢?

**Comment by woohsin** [comment]
參加系上活動有些都可以賺~~

**Comment by tzurou** [comment]
如果不排斥交友的話可以去參加營隊，我自己是在大一的時候加了友會的返服，大家一起努力辦一個活動真的很充實很快樂，我自己是i人，真的很慶幸當初有堅持下去沒有退出，也因為這樣交到一群很棒的朋友，現在大三了大家還是會偶爾聚在一起吃飯、唱歌，給你參考~

**Comment by EPHMR0524** [comment]
我非常推薦選修社會參與學分學程，這門課老師會帶著修個同學上山下海到處參觀，還會有校外服務時數可以拿，尤其是一定要跟隨蔡錫錚老師，他會對你所做的氣話給出很多的建議，在過程中除了能學習到一些做事的技巧，還有如何面對不講理的要求，我這門學程修過了1.5個學年，我覺得獲益良多，非常推薦大家選修並同時拿校外服務時數。

**Comment by NcuJerryKao** [comment]
這篇文章探討了學生取得校外服務時數的困難和經驗分享。以下是幾個建設性建議，以提升文章的結構、內容和清晰度：
1. 引言部分的強化：
- 引言可以更具信息性和吸引力，簡要介紹校外服務時數的重要性和取得的困難，讓讀者明白接下來討論的重點。
- 例如：「校外服務時數是學生服務時數中最難取得的項目之一，因為相關活動本來就不多，再加上申請手續繁雜，使得許多學生感到困擾。」
2. 段落結構：
- 將文章分成更易於理解的段落，每個段落集中討論一個具體的觀點。例如：
1. 引言：介紹校外服務時數的重要性和困難。
2. 個人經驗：分享參加淨灘活動的經歷和感受。
3. 校外服務時數的挑戰：探討申請手續繁雜和活動稀少的問題。
4. 解決方案和建議：提供一些取得校外服務時數的建議。
5. 結論和互動：總結文章並邀請讀者分享他們的經驗。
3. 過渡和連接詞：
- 使用過渡句和連接詞來連接不同段落，使文章流暢。例如，「接下來，我將分享我參加淨灘活動的經歷。」或「然而，取得校外服務時數的挑戰並不僅僅在於活動的稀少。」
4. 詳細說明和分析：
- 在描述參加淨灘活動的經歷時，可以更詳細地解釋其背後的意義和具體過程。例如，詳細說明活動的組織方式、參與者的反應以及對環境的影響。
- 例如：「參加淨灘活動時，我和同學們一起到海邊撿拾垃圾。這不僅讓我們回憶起童年的美好時光，還感受到戶外教學的樂趣。同時，我們也為保護環境出了一份力，讓海灘更加乾淨。」
5. 語言簡化和澄清：
- 某些段落中的語言較為簡單，可以考慮稍微豐富和澄清。例如，「校外服務時數的活動本來就不多」可以改為「校外服務時數的活動相對較少，學生很難找到合適的機會。」
6. 引用和支持：
- 若能引用一些具體的數據或專家觀點來支持論點，將大大增強文章的權威性。例如，可以引用教育部門的規定或相關研究來支持「校外服務時數的重要性」和「申請手續繁雜」的討論。
7. 個人經驗和反思：
- 個人經驗和反思是文章的一大亮點，可以更詳細地描述實際經歷和感受，這樣可以讓讀者更容易產生共鳴。
- 例如，「雖然參加淨灘活動讓我感到非常滿足，但交通不便確實是一個挑戰。希望未來能有更多便利的校外服務活動供學生選擇。」
8. 結論和建議強化：
- 在結論部分，再次強調文章的主要觀點，並總結文章的重要結論。這有助於讀者記住文章的核心思想。
- 例如：「總結來說，取得校外服務時數雖然困難，但通過積極參與活動和尋找合適的機會，我們仍能實現目標。希望未來能有更多方便的校外服務活動，讓學生更容易參與。」
9. 讀者互動：
- 在文章結尾，可以提出一些問題或邀請讀者分享他們的觀點和經驗，以增加互動性和參與感。
- 例如，「你有哪些取得校外服務時數的經驗或建議？你認為如何改進校外服務活動的申請流程？歡迎在評論區分享你的看法。」
通過這些建議，文章將變得更加結構化、清晰且具吸引力，讀者也會更容易理解和產生共鳴。

**Reply by yubo0oo** [reply]
用太誇張了吧烙葛

**Reply by cristal2466** [reply]
哈哈哈哈

**Comment by LinChloe** [comment]
去汪汪社溜狗，去幫舉辦的比賽當工人，土木的服學友給時數。
其實獲得時數的方式百百種，只是校外的都需要付出時間和勞力，所以大家都不太想去而已。
我也覺得參加營隊很麻煩，所以都沒去。

**Comment by hbh10601** [comment]
open house day 學系博覽會可以領8小時，其實就只是講解你系上在學什麼而已。
比較累的地方在於如果是熱門科系，你要一直和家長講話，會講到有點喉嚨痛。

**Comment by SusuTima** [comment]
可以去喵喵社喂貓，他們是排班制，可以撸貓順便賺時數。雖然最近有許多人對喵喵社有質疑，不過不論如何，我都希望學校的貓咪能有安全的環境生長。

**Comment by alexender0104** [comment]
我自己是在寒假的時候找尋區公所圖書館的志工活動，學校校外時數要求不多一個禮拜就可以完成。

**Comment by 155Julia** [comment]
我有一位打排球的朋友，因為他認識了相關的人脈，每當有中央舉辦的大專排球盃時，我就可以去當志工。這樣我不用到校外，就能賺取校外服務學習時數，真的很棒！我也推薦大家可以嘗試看看！

**Comment by OG-Matcha** [comment]
我自己是之前體育老師剛好有負責學校運動會的項目，就剛好可以去參加志工，寫個簡單的心得18小時校外時數就拿到了，還蹭了兩天的便當，賺爛

**Comment by OHoln** [comment]
我自己是參加系上的服務學習課程，一學期課程下來校外時數就直接滿了，基本上不用煩惱這個問題。

**Comment by wanka2030** [comment]
賺取校外時數可以通過多種方式進行。首先，參加志願者活動，如社區服務或非營利組織的項目，這不僅有助於積累時數，還能提升社會責任感。其次，尋找實習機會，尤其是與專業相關的實習，能為未來職業發展提供實踐經驗。此外，參加校外講座、工作坊和研討會也能計入時數，同時拓展知識面和人脈。總之，多參與各類有意義的校外活動，有助於全面發展並順利完成時數要求。

**Comment by tshsu04224** [comment]
目前大二也還沒賺滿校外時數，之前參加系上活動，像是宿營及畢業專題發表，本來都說會有校外時數，結果參加完卻申請不到🥲現在只有學系博覽會的八小時，還在想另外兩小時要做些什麼，淨灘的話，大一服務學習課程參加過，但算的卻是服務學習課程的時數，好難🫠

**Comment by 110101501** [comment]
校外時數的話，我自己是透過參加營隊，擔任營隊工人獲得。
光跑一次營隊，除了校外時數外，也可以獲得其他時數，例如生活知能。
身邊很多朋友光跑一次營隊就幾乎塊拿滿畢業所需的時數了。

**Comment by cassy0201** [comment]
通常參加一些服務性質的營隊或社團很快就可以拿滿了!

**Comment by 710388** [comment]
一律推薦喵喵社，照顧動物還可以賺時數

**Comment by yubo0oo** [comment]
我去潛水社淨灘
很有意義且可以拿到校外時數的一個好活動

**Comment by YeeDaniel** [comment]
個人推薦汪汪社和喵喵社，在嚕狗或嚕貓的過程也能順便攢點時數，也蠻不錯的~
還有淨灘活動也蠻推的，可以認識新朋友，也可以當作出遊的機會

**Comment by buoioiry** [comment]
有時間的話我覺得參加營隊是不錯的選擇，時數一下就滿了，不過有些人很討厭參與營隊活動，或是汪汪社那種服務性社團也可以賺時數，還可以碰到狗狗貓貓超可愛

**Comment by sunset513** [comment]
我的經驗是去當運動會的志工。

我當運動會志工的時候有點辛苦，會需要待再炎熱的天氣底下幾天。不過我除了獲得時數之外還有拿到一些好處，像是免費的便當，蠻開心的。

然後除了當志工幫忙做事外，結束後我還有些了一篇心得才可以拿到時數。

個人的小分享，希望對大家有幫助。

**Comment by Boying12** [comment]
校外服務時數的獲取途徑多樣而富有意義，學生們可以參與自己社區的志工服務，並向社區領導如里長申請證明書，以便向學校兌換相應的服務時數。此外，參加「汪汪社」的活動，帶著可愛的狗狗散步，不僅樂趣無窮，更能體會成就感，同時獲得充足的校外服務時數。除此之外，學校的友會活動也提供了與小朋友互動的機會，讓學生在與孩子們共度愉快時光的同時，獲得寶貴的校外服務時數。這些活動不僅充實了學生的課外生活，更培養了他們的社會責任感與人際交往能力。
